src/main/python/api.py

```diff
@@ -52,7 +53,11 @@ def get_answers(question, num_hits, k):
     parser.add_argument("--debug", help="print debug info", action="store_true")
     args = parser.parse_args()
 
-    config = ConfigParser.ConfigParser()
+    if not os.path.isfile(args.config):
+        print("The configuration file ({}) does not exist!".format(args.config))
+        exit(1)
```

Comment:
Should use `sys.exit` instead I think. See http://stackoverflow.com/a/6501134/2514889 for more details.